src/main/scala/beam/analysis/RoutingRequestAnalysis.scala

```diff
@@ -0,0 +1,59 @@
+package beam.analysis
+
+import beam.agentsim.events.ModeChoiceEvent
+import beam.analysis.plots.GraphAnalysis
+import beam.physsim.cchRoutingAssignment.OD
+import beam.router.BeamRouter.RoutingRequest
+import beam.router.r5.RouteDumper.RoutingRequestEvent
+import beam.sim.config.BeamConfig
+import beam.utils.FileUtils
+import beam.utils.csv.CsvWriter
+import org.matsim.api.core.v01.events.Event
+import org.matsim.core.controler.events.IterationEndsEvent
+import org.matsim.core.utils.io.IOUtils
+
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+case class CsvRow(personId: String, modeChoice: String, routingRequests: Vector[Int]) {
+  override def toString: String = s"$personId,$modeChoice,${routingRequests.mkString(",")}"
```

Comment:
It seems that `routingRequests` should be escaped otherwise CSV reader will try to split by `,` and assume that it is new column

Reply:
Put routing request ids in escaped characters